### DISCUSSION TITLE
NX-OS/netflow 9 exporters missing in the UI

### DISCUSSION THREAD

**Discussion by vvas1lev** [akvorado/akvorado#1129] [Q&A]
Hi all,

I've got a few NX-OS devices (configuration below), which don't show up in the UI. Akvorado is working perfectly fine for devices running IOS-XE, so the issue is limited to NX-OS.

=============================================
flow exporter AkvoradoExporter
  destination X.X.X.X use-vrf X
  transport udp 2055
  source Vlan235
  version 9
    template data timeout 20

flow record AkvoradoRecord
  match ipv4 source address
  match ipv4 destination address
  match ip protocol
  match ip tos
  match transport source-port
  match transport destination-port
  collect counter bytes long
  collect counter packets long
  collect timestamp sys-uptime first
  collect timestamp sys-uptime last
  collect ip version

sampler 1-out-of-1024
  mode 1 out-of 1024

flow monitor AkvoradoMonitor
  record AkvoradoRecord
  exporter AkvoradoExporter

interface Vlan235
  ip flow monitor AkvoradoMonitor input sampler 1-out-of-1024

Akvorado is receiving netflow packets:

 curl -s https://akvorado/api/v0/inlet/metrics | grep '^akvorado_inlet_flow_input_udp_packets'  | grep 235.2
akvorado_inlet_flow_input_udp_packets_total{exporter="X.X.X.X",listener=":2055",worker="1"} 11597

**Comment by fatman00** [comment]
Hi @vvas1lev,

Have you looked to see if there are any errors in the received packets?

`$ curl -s http://akvorado/api/v0/inlet/metrics | grep '^akvorado_inlet' | grep _errors`

**Comment by vvas1lev** [comment]
Yes, I have, and there's nothing with regards to the NX-OS hosts. I see these only:

akvorado_inlet_core_flows_errors_total{error="SNMP cache miss",exporter="Y.Y.Y.Y"} 24
akvorado_inlet_flow_decoder_netflow_errors_total{error="NetFlow/IPFIX decoding error",exporter="Y.Y.Y.Y"} 145

but Y.Y.Y.Y is an IOS-XE host. Everything works as expected for IOS-XE.

**Reply by fatman00** [reply]
I am not sure if you are using docker or not, but do you get any errors if you look at the inlet log?

`docker compose logs -f akvorado-inlet`
